src/components/Lesson/LessonList/index.tsx

```diff
@@ -1,43 +1,48 @@
-import React, { FC } from 'react';
-import { FlatList, TouchableOpacity } from 'react-native-gesture-handler';
+import React, { FC, memo } from 'react';
+import { FlatList } from 'react-native-gesture-handler';
 import { HorizontalLessonCard } from '../HorizontalLessonCard';
 import { VerticalLessonCard } from '../VerticalLessonCard';
 import useConnect from './connect';
 import { HorizontalView, VerticalView } from './styles';
 import { Props } from './types';
 
-export const LessonList: FC<Props> = ({ categoryFlag }) => {
+export const LessonList: FC<Props> = ({ isFiltered, lessons }) => {
```

Comment:
isFiltered is a String, not a boolean. We are very strict with namings. 
Regarding the name of the variable, you should change the name to something more grammarly correct for variable which is a string. We should use "is-" or "has-" only for booleans.

```suggestion
export const LessonList: FC<Props> = ({ filter, lessons }) => {
```

Reply:
Noted!